src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -77,12 +87,20 @@ public VisualEditorPageObject(WebDriver driver) {
   private WebElement mainContent;
   @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode")
   private WebElement galleryNode;
-  @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode>div")
-  private List<WebElement> galleryNodes;
   @FindBy(css = ".media-gallery-wrapper.ve-ce-branchNode .toggler")
   private WebElement toggler;
   @FindBy(css = ".ve-ce-surface-highlights-focused .ve-ce-focusableNode-highlight")
   private WebElement focusedHighlight;
+  @FindBy(css = ".ve-init-mw-viewPageTarget-surface")
+  private WebElement veSurface;
+  @FindBy(css = ".oo-ui-popupWidget-body .oo-ui-widget-enabled")
+  private WebElement infoboxPopup;
+  @FindBy(css = ".oo-ui-processDialog-actions-primary .oo-ui-buttonElement")
+  private WebElement applyChangesButton;
+  @FindBy(css = ".oo-ui-labelElement.oo-ui-popupToolGroup.oo-ui-listToolGroup")
+  private WebElement insertDropdownMenuButton;
+  @FindBy(css = ".oo-ui-toolGroup-tools .oo-ui-icon-infobox")
+  private WebElement infoboxInDropdownMenu;
 
   private By contextMenuBy = By.cssSelector(".ve-ui-contextSelectWidget");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)